src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -0,0 +1,235 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <bitmap name="ScenesCopyMode" type="BITMAP8">
+    <cluster code="0x0005"/>
+    <field name="CopyAllScenes" mask="0x01"/>
+  </bitmap>
+
+  <struct name="AttributeValuePair">
+    <cluster code="0x0005"/>
+    <item name="attributeId" type="attrib_id" optional="true"/>
+    <item name="attributeValue" type="INT8U" array="true"/>
+  </struct>
+
+  <struct name="SceneExtensionFieldSet">
+    <cluster code="0x0005"/>
+    <item name="clusterId" type="cluster_id"/>
+    <item name="attributeValueList" type="AttributeValuePair" array="true"/>
+  </struct>
+
+  <domain name="CHIP"/>
+    <cluster>
+    <name>Scenes</name>
+    <domain>General</domain>
+    <description>Attributes and commands for scene configuration and manipulation.</description>
+    <code>0x0005</code>
+    <define>SCENES_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <globalAttribute side="either" code="0xFFFD" value="4"/>
+   
+    <attribute side="server" code="0x0000" define="SCENE_COUNT" type="INT8U" min="0x00" max="0xFF" writable="false" default="0x00" optional="false">sceneCount</attribute>
+    <attribute side="server" code="0x0001" define="CURRENT_SCENE" type="INT8U" min="0x00" max="0xFF" writable="false" default="0x00" optional="false">currentScene</attribute>
+    <attribute side="server" code="0x0002" define="CURRENT_GROUP" type="group_id" min="0x0000" max="0xFFF7" writable="false" default="0x0000" optional="false">currentGroup</attribute>
+    <attribute side="server" code="0x0003" define="SCENE_VALID" type="BOOLEAN" min="0x00" max="0x01" writable="false" default="0x00" optional="false">sceneValid</attribute>
+    <attribute side="server" code="0x0004" define="SCENE_NAME_SUPPORT" type="BITMAP8" min="0x00" max="0x80" writable="false" default="0x00" optional="false">nameSupport</attribute>
+    <attribute side="server" code="0x0005" define="LAST_CONFIGURED_BY" type="node_id" writable="false" isNullable="true" optional="true">lastConfiguredBy</attribute>
+   
+    <command source="client" code="0x00" name="AddScene" response="AddSceneResponse" optional="false" cli="chip scenes add">
+      <description>
+        Add a scene to the scene table. Extension field sets are supported, and are inputed as arrays of the form [[cluster ID] [length] [value0...n] ...]
+      </description>
+      <arg name="groupId" type="group_id"/>
```

Comment:
Again, the names should match the spec and be "GroupId" and the like.  I'm going to stop commenting on that part.